study.md

```diff
@@ -40,7 +41,9 @@ In your own words, define what the responsibilities of the controller layer are
 in Rails.
 
 ```md
-<!-- your answer here -->
+The controller layer deals with user input.  It receives input and decides what to
+do with it.  Ideally you want your controller to take inputs, call model methods and
+pass outputs.
```

Comment:
That's right. These outputs usually become part of the response that Rails generates implicitly after each controller action.